src/components/header.js

```diff
@@ -1,20 +1,57 @@
-import PropTypes from 'prop-types'
 import React from 'react'
-const Header = () => {
+import { Link } from 'gatsby'
+import Image from '../components/image'
+import { makeStyles } from '@material-ui/core/styles'
+import AppBar from '@material-ui/core/AppBar'
+import Toolbar from '@material-ui/core/Toolbar'
+import Button from '@material-ui/core/Button'
+import Container from '@material-ui/core/Container'
 
-  return (
-    <header>
-    <p>header</p>
-    </header>
-  )
-}
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+  },
+  appBarHeader: {
+    boxShadow: 'none',
+  },
+  logo: {
+    flexGrow: 1,
+    margin: theme.spacing(1),
+    maxWidth: '60px',
+  },
+  nav: {
+    position: 'absolute',
+    right: '10px'
+  },
+  navButton: {
+    margin: theme.spacing(1),
+  },
+}));
 
-Header.propTypes = {
-  siteTitle: PropTypes.string,
-}
+const Header = () => {
+  const classes = useStyles();
 
-Header.defaultProps = {
-  siteTitle: ``,
+  return (
+    <div className={classes.root}>
+      <Container component='div'>
+        <AppBar position="static" color="transparent" className={classes.appBarHeader}>
+          <Toolbar>
+            <div className={classes.logo}>
+              <Image />
+            </div>
+            <div className={classes.nav}>
+              <Link to="/">
```

Comment:
Can we add an `activeStyle` to there links so we can show the user the link they are currently on. 

Also the Link component creates an '<a>' tag which then is wrapping around the '<button>' tag which isn't valid HTML, can we change the button to be just a 'p' tag?

Reply:
ah yes activeStyle is a good point did not think. about it.

and with you tag point this is a very valid point and will def change this, will just check how to get that material-ui button effect on the p tag! unless you don't think it is necessary.

We get on these changes a little later sorry for the delay.